### ISSUE TITLE
WAN access suddenly stopped

### ISSUE THREAD

**Issue by chadwagoner-sf** [lloesche/valheim-server-docker#561]
In the past day or 2 (Jan 7-8 timeframe), public IP access suddenly stopped with no real reason why.. Just 'failed to connect' message when trying to connect..

My setup is pretty simple, using base game (no mods) and the follow variable file..

```
SERVER_NAME=<server-name>
WORLD_NAME=<workd-name>
SERVER_PASS=<server-password>
SERVER_PUBLIC=true
```

After messing with the container all day, I added `SERVER_ARGS=-crossplay` and it seems to have fixed my issue..

Not sure if this is related to #552 but wanted to share my discovery in case anyone else has experienced this as of late randomly..

**Comment by wcdanh2** [comment]
Have you verified that your public ip address hasn't changed?

**Comment by chadwagoner-sf** [comment]
To add, from what I could tell in the logs aka stdout from container.. It referenced query port 2457/udp and that is about now.. After adding the `-crossplay` flag, I see much more logging around playfab and resulting with `Session "<server-name>" with join code <join-code> and IP <wan-ip>:2456 is active with 0 player(s)`

**Comment by chadwagoner-sf** [comment]
Nope, WAN ip does not change as I have a static ip.. It seemed to have randomly stopped working very recently and started to dig into it today.. This isn't as much of an 'issue' as a headups/awareness
